### PR TITLE
chore: add fast MSFT component version to explore site

### LINKED ISSUE #2257: Add build info to all FAST apps
**What feature are you requesting?**
Let's have some way to see basic build information in the tools; at least the build date, but possibly a build number if that's useful, or version numbers of component libraries.

**What would be the benefit of adding this feature?**
As the number of apps and deployment location grows, it's increasingly harder to know what's current and how current it is. We need to know that if we direct a partner somewhere, that it's up to date.

**What solution would be ideal for you?**
Any of these options would be a good start:
- A url that's not even linked or indexed, like /ver.txt
- A tooltip over the product header
image
- A question mark icon in that same space, with tooltip or more detailed flyout

### PR DESCRIPTION
**Pull request by chrisdholt**
# Description
This PR enables the ability to dynamically add version information for the MSFT components to the explore site. 
**Edit: I've update this to use the title attribute rather than passing the version as part of the heading itself.**

## Motivation & context
closes #2257 

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-component-explorer/app/explorer.tsx [hunk 1]
```diff
@@ -126,6 +128,8 @@ class Explorer extends Foundation<
 > {
     public static displayName: string = "Explorer";
 
+    public static version: string = version;
```
Comment (janechu) [inline packages/fast-component-explorer/app/explorer.tsx]: does this need to be added? seems like we're just using the imported version

File: packages/fast-component-explorer/app/explorer.tsx [hunk 2]
```diff
@@ -321,7 +325,12 @@ class Explorer extends Foundation<
                         >
                             <Background value={neutralLayerL3} drawBackground={false}>
                                 <div className={classNames(explorer_paneTitleContainer)}>
-                                    <Heading size={HeadingSize._6}>FAST Explorer</Heading>
+                                    <Heading
+                                        size={HeadingSize._6}
+                                        title={`Version ${version}`}
```
Comment (janechu) [inline packages/fast-component-explorer/app/explorer.tsx]: should we consider using `Badge`?

Comment (chrisdholt) [reply packages/fast-component-explorer/app/explorer.tsx]: We could add Badge - I meant to open as a draft initially, but here we are :)

Comment (bheston) [reply packages/fast-component-explorer/app/explorer.tsx]: That would be consistent with what we did for Creator. I think that works well considering the related change to publish the latest published version.
image

File: packages/fast-component-explorer/app/explorer.tsx [hunk 3]
```diff
@@ -321,7 +325,12 @@ class Explorer extends Foundation<
                         >
                             <Background value={neutralLayerL3} drawBackground={false}>
                                 <div className={classNames(explorer_paneTitleContainer)}>
-                                    <Heading size={HeadingSize._6}>FAST Explorer</Heading>
+                                    <Heading
```
Comment (nicholasrice) [inline packages/fast-component-explorer/app/explorer.tsx]: We discussed somehow denoting that these changes could be pre-release and not available via npm in the staging case - is that work going into this PR?

Comment (chrisdholt) [reply packages/fast-component-explorer/app/explorer.tsx]: I'm working on that as part of this PR.

**Pull request closed without merging**